.github/workflows/ci.yml

```diff
@@ -5,143 +5,314 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+env:
+  DEFAULT_NODE_VERSION: 22.x
+  DOCUMENTATION_BRANCH: feat/simplify-extension-ignores
```

Comment:
Is this correct?

Reply:
The Node version yes, the docs branch should not be even used anywhere anymore, it is a remainder from my WIP branch. I need to double-check this, thank you for spotting it! :grimacing: